package com.example.todo.config

import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@EnableConfigurationProperties(
    AppProperties::class,
    JwtProperties::class,
    AuthProperties::class
)
class PropertiesConfig